zapier/introduction/zaps-ii/zapier-data.md

```diff
@@ -0,0 +1,37 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# Zapier Data
+
+---
+## Content
+
+Until now, we have only used simple actions in our zaps (monitoring some data or sending a message). Zapier is not limited to that, with another popular application being finding data and using it at a later time.
+
+We have somewhat done this, when we used the filter step to verify that someone has actually gained access to our base.
+
+Now, what if we want to know who has gained access to our base? This is entirely possible, and since we are using Google Sheets we don't require any extra steps.
+
+All that you would have to do is go to the Slack step, and in the `'Message Text'` box you can already access all the information seen in the linked spreadsheet.
+
+![gif-of-data](#)
+
+![image-of-completed-data](https://img.enkipro.com/936b5a4e98e90a855f51334425e4fd5f.png)
+
+---
+## Practice
+
+With Zapier, you can find data in an app and ???.
```

Comment:
the answer to this question is pretty much gave away through elimination

thinking about it this insight doesn't introduce anything new that could be asked to I don't really have a better suggestion :/

Reply:
maybe we could have a multiple choice thing wheret they need to pick what can be done with zapier:
- monitor data and react to changes
- access data which was reacted to at a later time
- send messages
- use it as a database
- replace note taking apps 
etc